docs/layout/spacing/README.md

```diff
@@ -0,0 +1,205 @@
+| cdr-space-one-x            | 16px         | 1.6rem     |
+| cdr-space-one-and-a-half-x | 24px         | 2.4rem     |
+| cdr-space-two-x            | 32px         | 3.2rem     |
+| cdr-space-four-x           | 64px         | 6.4rem     |
+
+
+### Spacing Example
+
+An example of setting the generic space classes:
+
+| Modifier      | CSS property                 | Space utility class           |
+|---------------|------------------------------|-------------------------------|
+| pt            | padding-top                  | cdr-pt-space-one-x            |
+| pr            | padding-right                | cdr-pr-space-one-x            |
+| pb            | padding-bottom               | cdr-pb-space-one-x            |
+| pl            | padding-left                 | cdr-pl-space-one-x            |
+| px            | padding-left & padding-right | cdr-px-space-one-x            |
+| py            | padding-top & padding-bottom | cdr-py-space-one-x            |
+| mt            | margin-top                   | cdr-mt-space-one-x            |
+| mr            | margin-right                 | cdr-mr-space-one-x            |
+| mb            | margin-bottom                | cdr-mb-space-one-x            |
+| ml            | margin-left                  | cdr-ml-space-one-x            |
+| mx            | margin-left & margin-right   | cdr-mx-space-one-x            |
+| my            | margin-top & margin-bottom   | cdr-my-space-one-x            |
+| inset         | padding                      | cdr-space-inset-one-x         |
+| inset-squish  | padding                      | cdr-space-inset-squish-one-x  |
+| inset-stretch | padding                      | cdr-space-inset-stretch-one-x |
+
+For a full list of space utility classes, see the [CSS Utility Class](../../components/utilities/) document.
+
+### Breakpoint (Optional)
+
+Sometimes your content may need to be spaced differently at specific breakpoints. To accommodate such different spacing needs, spacing utilities are available at each of our defined breakpoints by concatenating the ‘@’ symbol with the desired breakpoint to the end of the class name.
+
+-  `xs` - applies the spacing **for the extra small breakpoint ( < 768px)**
+-  `sm` - applies the spacing **for the small breakpoint (768px <= X < 992px)** 
+-  `md` - applies the spacing **for the medium breakpoint (992px <= X < 1232px)**
+-  `lg `- applies the spacing **for the large breakpoint ( > 1232px)**
+
+For example, `.cdr-space-inset-eighth-x@md` will apply if the user’s screen is currently at the medium breakpoint (within 992px and 1232px wide).
+
+For more information on breakpoints and responsive design, see the [Responsive Layout](../responsive) article.
+
+To add right margin to the Cedar button component at the medium breakpoint and up, but bottom margin to at the small breakpoint and below:
+
+
+For a full list of space utility classes, see the [CSS Utility Class](../../components/utilities/) document.
+
```

Comment:
This is the last sentence of the article.

Reply:
But, I think we can delete this sentence from here, as we are already adding it at the end.